iib/workers/tasks/utils.py

```diff
@@ -91,11 +91,19 @@ def add_max_ocp_version_property(resolved_bundles, temp_dir):
         return
 
     # Filter index image bundles to get pull spec for bundles in the request
-    updated_bundles = list(
-        filter(lambda b: b['bundlePath'] in resolved_bundles, get_bundle_json(raw_bundles))
-    )
+    present_bundles = get_bundle_json(raw_bundles)
```

Comment:
May I suggest adding a test and description in the commit/PR for this change?

Reply:
Yep, that would be nice. Otherwise it looks OK.

Reply:
I agree with adding a test to make sure the bundles are unique and functions like `_requires_max_ocp_version` and `_add_property_to_index` are called as expected.

Reply:
I don't think we get duplicate bundles when fetching bundles from the index. Because OPM doesn't allow adding duplicates in the first place.